README.md

```diff
@@ -1,62 +1,86 @@
+[![](https://user-images.githubusercontent.com/1248214/57789522-600fcc00-7739-11e9-86d9-73d7032f40fc.png)
```

Comment:
General thoughts: this is so much better!   

Just to make sure that we keep track of the other ideas we had to improve this README (e.g. contribution guidelines, social channels etc) and that are not part of this iteration: maybe we could create a ticket with these ideas listed?

Reply:
Added more in the documentation board.